<ui version="4.0" >
 <class>AIdePrefDialogClass</class>
 <widget class="QDialog" name="AIdePrefDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>IDE Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="upSettingsTab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Editor" >
      <attribute name="title" >
       <string>Editor</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="upEditorFontLabel" >
           <property name="text" >
            <string>Font:</string>
           </property>
           <property name="buddy" >
            <cstring>upEditorFont</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="upEditorFont" >
           <property name="fontFilters" >
            <set>QFontComboBox::MonospacedFonts</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="upEditorFontSizeLabel" >
           <property name="text" >
            <string>Size:</string>
           </property>
           <property name="buddy" >
            <cstring>upEditorFontSize</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="upEditorFontSize" >
           <property name="minimum" >
            <number>6</number>
           </property>
           <property name="maximum" >
            <number>24</number>
           </property>
           <property name="singleStep" >
            <number>2</number>
           </property>
           <property name="value" >
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="upEditorTabWidthLabel" >
           <property name="text" >
            <string>Tab Width:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="upEditorTabWidth" >
           <property name="minimum" >
            <number>2</number>
           </property>
           <property name="maximum" >
            <number>16</number>
           </property>
           <property name="singleStep" >
            <number>2</number>
           </property>
           <property name="value" >
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="upEditorLineNumbers" >
         <property name="text" >
          <string>Show Line Numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upEditorWhiteSpaces" >
         <property name="text" >
          <string>Show White Spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upEditorWordWrap" >
         <property name="text" >
          <string>Word Wrap</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Debugger" >
      <attribute name="title" >
       <string>Debugger</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="upDebuggerFontLabel" >
           <property name="text" >
            <string>Font:</string>
           </property>
           <property name="buddy" >
            <cstring>upDebuggerFont</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="upDebuggerFont" >
           <property name="fontFilters" >
            <set>QFontComboBox::MonospacedFonts</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="upDebuggerFontSizeLabel" >
           <property name="text" >
            <string>Size:</string>
           </property>
           <property name="buddy" >
            <cstring>upDebuggerFontSize</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="upDebuggerFontSize" >
           <property name="minimum" >
            <number>6</number>
           </property>
           <property name="maximum" >
            <number>24</number>
           </property>
           <property name="singleStep" >
            <number>2</number>
           </property>
           <property name="value" >
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="upDebuggerTabWidthLabel" >
           <property name="text" >
            <string>Tab Width:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="upDebuggerTabWidth" >
           <property name="minimum" >
            <number>2</number>
           </property>
           <property name="maximum" >
            <number>16</number>
           </property>
           <property name="singleStep" >
            <number>2</number>
           </property>
           <property name="value" >
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Console" >
      <attribute name="title" >
       <string>Console</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="upConsoleFontLabel" >
           <property name="text" >
            <string>Font:</string>
           </property>
           <property name="buddy" >
            <cstring>upEditorFont</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="upConsoleFont" >
           <property name="fontFilters" >
            <set>QFontComboBox::MonospacedFonts</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="upConsoleFontSizeLabel" >
           <property name="text" >
            <string>Size:</string>
           </property>
           <property name="buddy" >
            <cstring>upEditorFontSize</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="upConsoleFontSize" >
           <property name="minimum" >
            <number>6</number>
           </property>
           <property name="maximum" >
            <number>24</number>
           </property>
           <property name="singleStep" >
            <number>2</number>
           </property>
           <property name="value" >
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="upConsoleTabWidthLabel" >
           <property name="text" >
            <string>Tab Width:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="upConsoleTabWidth" >
           <property name="minimum" >
            <number>2</number>
           </property>
           <property name="maximum" >
            <number>16</number>
           </property>
           <property name="singleStep" >
            <number>2</number>
           </property>
           <property name="value" >
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="upConsoleLineNumbers" >
         <property name="text" >
          <string>Show Line Numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upConsoleWhiteSpaces" >
         <property name="text" >
          <string>Show White Spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upConsoleWordWrap" >
         <property name="text" >
          <string>Word Wrap</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Log" >
      <attribute name="title" >
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="upLogFontLabel" >
           <property name="text" >
            <string>Font:</string>
           </property>
           <property name="buddy" >
            <cstring>upEditorFont</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="upLogFont" >
           <property name="fontFilters" >
            <set>QFontComboBox::MonospacedFonts</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="upLogFontSizeLabel" >
           <property name="text" >
            <string>Size:</string>
           </property>
           <property name="buddy" >
            <cstring>upEditorFontSize</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="upLogFontSize" >
           <property name="minimum" >
            <number>6</number>
           </property>
           <property name="maximum" >
            <number>24</number>
           </property>
           <property name="singleStep" >
            <number>2</number>
           </property>
           <property name="value" >
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="upLogLineNumbers" >
         <property name="text" >
          <string>Show Line Numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="upLogWordWrap" >
         <property name="text" >
          <string>Word Wrap</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Others" >
      <attribute name="title" >
       <string>Others</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="upCabinetGroup" >
         <property name="title" >
          <string>Cabinet Page</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="upCabinetFontLabel" >
              <property name="text" >
               <string>Font:</string>
              </property>
              <property name="buddy" >
               <cstring>upEditorFont</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="upCabFont" >
              <property name="fontFilters" >
               <set>QFontComboBox::AllFonts</set>
              </property>
              <property name="currentFont" >
               <font>
                <family>DejaVu Sans</family>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="upCabFontSizeLabel" >
              <property name="text" >
               <string>Size:</string>
              </property>
              <property name="buddy" >
               <cstring>upEditorFontSize</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="upCabFontSize" >
              <property name="minimum" >
               <number>6</number>
              </property>
              <property name="maximum" >
               <number>24</number>
              </property>
              <property name="singleStep" >
               <number>2</number>
              </property>
              <property name="value" >
               <number>12</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upOkButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>upSettingsTab</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>upSettingsTab</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
